bytecode/instr.py

```diff
@@ -282,6 +282,37 @@ def stack_effect(self, jump=None):
         else:
             return dis.stack_effect(self._opcode, arg, jump=jump)
 
+    def pushes_back(self):
```

Comment:
Should have a leading underscore, this is meant to be private. And it does not need to be a method, we could extract the opcode name once, and pass it to an independent function, but this should be fine.